stage1/init/init.go

```diff
@@ -577,16 +577,17 @@ func stage1() int {
 		return 1
 	}
 
-	// create a separate mount namespace so the cgroup filesystems
-	// are unmounted when exiting the pod
+	// Creates a separate mount namespace so the cgroup filesystems
```

Comment:
I'm actually -1 on these changes - not every inline comment should be a fully formatted prose sentence, and I think the imperative style is preferable when describing the purpose of the code. @joshix thoughts?